api/api.go

```diff
@@ -391,13 +395,18 @@ func (b *BlockChainAPI) Call(
 	overrides *StateOverride,
 	blockOverrides *BlockOverrides,
 ) (hexutil.Bytes, error) {
+	cadenceHeight, err := b.getCadenceHeight(blockNumberOrHash)
+	if err != nil {
+		return nil, err
```

Comment:
all the errors should be wrapped in the `handleError`

Reply:
Good catch :100: Updated in https://github.com/onflow/flow-evm-gateway/pull/192/commits/75b458e89d194be525a227d6c022c2377803f00c